WordPressCom-Analytics-iOS.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "WordPressCom-Analytics-iOS"
-  s.version      = "0.0.36"
+  s.version      = "0.0.37"
```

Comment:
I'm not sure how this will merge with develop but, this should be `0.38.2` (as develop has it right now) or be left unchanged - so the merge doesn't change it.